Mathlib/SetTheory/Game/Basic.lean

```diff
@@ -338,28 +336,28 @@ theorem mul_moveRight_inr {x y : PGame} {i j} :
   rfl
 #align pgame.mul_move_right_inr SetTheory.PGame.mul_moveRight_inr
 
--- @[simp] -- Porting note: simpNF linter complains
+@[nolint simpNF, simp] -- Porting note: simpNF linter complains, but this is a useful dsimp lemma
```

Comment:
While we're looking for someone to review the mathematically important parts of the PR, could you please comment on what `simp` lemmas can simplify the LHS of this and other lemmas you mark `nolint simpNF, simp`? Are they `dsimp`?